user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+.. _user-guide-user-management-permissions-basic:
+
+Access and Permissions Basics
+=============================
+
+OroCRM provides a lot of functionality that can enhance your CRM process. These may be used by different members of the 
+team. However, it is obvious that information and tools used by a marketing associate are different from 
+those required by support teams, or by the company management. Flexible structure of the access and permissions settings 
+helps to ensure that all the information is safe and available right where it is required, and all the users gain access 
+to the features and capabilities meeting their needs.
+
+This guide is aimed to explain the basic ideas of the access and permission settings. If you are the system 
+administrator, please address the :ref:`Access and Permissions Management <user-guide-user-management-permissions>`
+guide.
+
+What Define the Permissions
+---------------------------
+
+Access and permission rights of each user depend on the following: 
+
+- Organizations and units, to which the user belongs: so users with the same role, working in different offices or 
+  subdivisions may have different access to different information. The list of organizations and units to which a user 
+  belongs is defined by the system administrator in the *"Access Settings"*. 
+  
+- Roles, assigned to the user: for example the sales representatives and system administrators need very different data 
+  and have different access to it. The roles of a user are defined by the system administrator for each user in the 
+  *"Groups and Role"*.
+
+- Ownership type of the entity: when you create a record, you may define its *"Owner"*: these may be the user who can 
+  manage the records or a business unit/organization the members whereof can manage the record. Some record types have 
+  the ownership type *"None"*, which means that all the users can see them. Ownership type of each entity is defined by 
+  the system administrator. 
+
+
+What You Can Be Permitted
+-------------------------
+
+Access to Functionalities
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users with specific roles may be allowed or not allowed to use specific functionality. For example, some users may not 
+have capability to assign tags, to share grids, or to create new users.
+
+These are defined by the system administrator in the *"Capabilities section"* of each role. Only users with a role, for 
+which a capability is enabled, will be able to use the functionality. 
+
+.. note::
+
+    Capabilities of different roles are compounded, i.e. if one role has the right to merge entities, the user will 
+    have this right, regardless of the settings defined for other roles of this user.
```

Comment:
Edit: Capabilities of different roles are compounded. For example, if one role has the right to merge entities, the user will have this right, regardless of the settings defined for other roles of this user.